examples/c++/risc0/compile.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Exit immediately on error
+set -e
+
+mkdir -p build/example
+mkdir -p build/icicle
+
+# Configure and build Icicle
+# cmake -S ../../../icicle/ -B build/icicle -DCMAKE_BUILD_TYPE=Release -DCURVE=bn254 -DG2=OFF
+cmake -S ../../../icicle/ -B build/icicle -DCMAKE_BUILD_TYPE=Release -DFIELD=babybear -DG2=OFF
```

Comment:
I would remove the G2 param which is only relevant to curves. Or replace with EXT_FIELD which could be on here.
Also please deelte the commented line

Reply:
I agree...please remove

Reply:
done